java/semantickernel-api/src/main/java/com/microsoft/semantickernel/semanticfunctions/KernelFunctionFromPrompt.java

```diff
@@ -275,6 +275,7 @@ public FromPromptBuilder withPromptTemplateFactory(
         @Override
         public KernelFunction build() {
             PromptTemplateConfig config = new PromptTemplateConfig(
+                1,
```

Comment:
Don't let @karianna see this.

Reply:
<Appears in Sulphur and Flames>What does `1` mean? :-)</Appears in Sulphur and Flames>